src/util/criaParametrosBusca.js

```diff
@@ -0,0 +1,34 @@
+function criaPropriedade(value) {
+  if (value) return (value = new RegExp(value.replace(',', '|'), 'i'));
```

Comment:
```suggestion
  if (value) return new RegExp(value.replace(',', '|'), 'i');
```

pelo que entendi, só precisa retornar o novo valor, né?

Reply:
perfeito.